src/client/src/stratigraphy/stratigraphy.js

```diff
@@ -51,7 +51,7 @@ const Stratigraphy = props => {
   useEffect(() => {
     setState(prevState => ({
       ...prevState,
-      selected,
+      selected: selected ?? null,
```

Comment:
Q: Also ich blicke hier mit dem  Datenflow nicht wirklich durch. So wie ich sehe wird der Stratigraphy Komponente nie das prop "selected" übergeben. Dann kann man es auch aus den props löschen und gleich im State mit null initialisieren. 
Mal davon abgesehen, dass hier das "alte" react setState verhalten mit dem "neuen" useState hook nachgebaut wurde  (??). Aber ist vielleicht auch grad nicht der Moment das zu ändern ;)

Reply:
Ich nehme an, dass der Ablauf etwa folgender ist: In der initialisierung von `useState` wird selected auf `null` gesetzt. `const selected` ist aber `undefined`, da es in den props nicht gesetzt wurde. Darum wird nach dem rendern der Effekt ausgeführt, der `selected` im state ändert.

Das `selected` prop finde ich auch etwas unnötig.